test/integration/private_zonal_with_networking/testdata/TestPrivateZonalWithNetworking.json

```diff
@@ -50,15 +51,10 @@
   "labelFingerprint": "a9dc16a7",
   "legacyAbac": {},
   "location": "us-east4-a",
-  "locations": [
-    "us-east4-a"
-  ],
+  "locations": ["us-east4-a"],
```

Comment:
Hi @Tolsee - Looks like the original formatting was specific:
```
Step #78 - "verify private-zonal-with-networking":         	Error:      	Not equal: 
Step #78 - "verify private-zonal-with-networking":         	            	expected: "[\"us-east4-a\"]"
Step #78 - "verify private-zonal-with-networking":         	            	actual  : "[\n    \"us-east4-a\"\n  ]"
Step #78 - "verify private-zonal-with-networking":         	            	
Step #78 - "verify private-zonal-with-networking":         	            	Diff:
Step #78 - "verify private-zonal-with-networking":         	            	--- Expected
Step #78 - "verify private-zonal-with-networking":         	            	+++ Actual
Step #78 - "verify private-zonal-with-networking":         	            	@@ -1 +1,3 @@
Step #78 - "verify private-zonal-with-networking":         	            	-["us-east4-a"]
Step #78 - "verify private-zonal-with-networking":         	            	+[
Step #78 - "verify private-zonal-with-networking":         	            	+    "us-east4-a"
Step #78 - "verify private-zonal-with-networking":         	            	+  ]
Step #78 - "verify private-zonal-with-networking":         	Test:       	TestPrivateZonalWithNetworking
Step #78 - "verify private-zonal-with-networking":         	Messages:   	expected locations to match fixture ["us-east4-a"]
```

```suggestion
  "locations": [
    "us-east4-a"
  ],
```

There is a good chance this is true for the rest of the entries, so I would suggest reverting everything except the new `"gkeBackupAgentConfig": {},`.

Reply:
Thanks updated.